src/services/api/usersApi/index.ts

```diff
@@ -10,8 +10,15 @@ const usersApi = {
   postSearchUser: (email: string): Promise<AxiosResponse<User>> =>
     apiPost("users/search", { email }),
 
-  getUserStatistics: (id: number): Promise<AxiosResponse<User>> =>
-    apiGet(`users/${id}/statistics`),
+  getUserStatistics: (
+    id?: number,
+    walletAddress?: string,
+  ): Promise<AxiosResponse<User>> =>
+    apiGet(
+      `users/statistics/?${id ? `id=${id}` : ""}${
+        walletAddress ? `&walletAddress=${walletAddress}` : ""
+      }`,
+    ),
```

Comment:
what about this way?

```suggestion
    apiGet(
      `users/statistics/?${id ? `id=${id}&` : ""}${
        walletAddress ? `walletAddress=${walletAddress}` : ""
      }`,
    ),
```

moving the `&` from the beginning of `wallerAddress` to the ending of the `id` parameter, we can prevent cases like:

`url.com/?&wallet_addess=0XAAAA`

Reply:
yesss, i don't think about thatt, thankss